shell/browser/native_window_views_win.cc

```diff
@@ -465,10 +465,11 @@ bool NativeWindowViews::PreHandleMSG(UINT message,
     }
     case WM_SYSCOMMAND: {
       // Mask is needed to account for double clicking title bar to maximize
```

Comment:
Overall lgtm, nit: can this comment be amend since it is now referencing a command rather than the previous mask?